hawk/src/main/java/com/orhanobut/hawk/DataHelper.java

```diff
@@ -40,57 +37,74 @@ static DataInfo getDataInfo(String storedText) {
     if (TextUtils.isEmpty(storedText)) {
       throw new NullPointerException("Text should not be null or empty");
     }
-    int index = storedText.indexOf(DELIMITER);
+
+    int index = storedText.indexOf(INFO_DELIMITER, 0);
```

Comment:
Yeah.. :D dont ask me why I wrote that

On Sun, 20 Dec 2015 11:05 Orhan Obut <notifications@github.com> wrote:

> In hawk/src/main/java/com/orhanobut/hawk/DataHelper.java
> <https://github.com/orhanobut/hawk/pull/122#discussion_r48102281>:
>
> > @@ -40,57 +37,74 @@ static DataInfo getDataInfo(String storedText) {
> >      if (TextUtils.isEmpty(storedText)) {
> >        throw new NullPointerException("Text should not be null or empty");
> >      }
> > -    int index = storedText.indexOf(DELIMITER);
> > +
> > +    int index = storedText.indexOf(INFO_DELIMITER, 0);
>
> as default, starting index is 0, could stay like this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/orhanobut/hawk/pull/122/files#r48102281>.
>